CMakeLists.txt

```diff
@@ -19,15 +22,23 @@ add_subdirectory(libuv)
 set_target_properties(uv_a PROPERTIES
         INTERFACE_INCLUDE_DIRECTORIES ${CMAKE_CURRENT_SOURCE_DIR}/libuv/include)
 
-add_library(uv_mbed STATIC
-        src/uv_mbed.c src/uv_mbed.c src/bio.c src/bio.h)
+if (WIN32)
```

Comment:
probably better to create a list of source files and use it in `add_library()` directive, conditionally append to the list.

See libuv/CMakeLists.txt for example